### ISSUE TITLE
Is there a plan to add loop closer to generate a map?

### ISSUE THREAD

**Issue by qpc001** [Huguet57/LIMO-Velo#25]

**Comment by Huguet57** [comment]
Yes and no, the branch "hdmaps" (still a bit in progress) allows to generate a map. But I have no plans yet to generate a good loop closing since it's a big change and have 0 time to allocate to this project now...
